docs/reference/mapping/types/sparse-vector.asciidoc

```diff
@@ -91,7 +91,7 @@ NOTE: `sparse_vector` fields can not be included in indices that were *created*
 NOTE: `sparse_vector` fields only support strictly positive values.
 Negative values will be rejected.
 
-NOTE: `sparse_vector` fields do not support querying, sorting or aggregating.
+NOTE: `sparse_vector` fields do not support <<analysis-intro,analyzers>>, querying, sorting or aggregating.
```

Comment:
```suggestion
NOTE: `sparse_vector` fields do not support <<analysis,analyzers>>, querying, sorting or aggregating.
```

Sorry, mixed up the ES "guide" with "reference" pages. Now will link to [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis.html).

Reply:
It can all get very 😵‍💫 fast :)